src/Factory/CalloutRuleFactory.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusCalloutsPlugin\Factory;
+
+use Setono\SyliusCalloutsPlugin\Checker\Rule\HasTaxonCalloutRuleChecker;
+use Setono\SyliusCalloutsPlugin\Model\CalloutRule;
+use Setono\SyliusCalloutsPlugin\Model\CalloutRuleInterface;
+use Sylius\Component\Core\Model\TaxonInterface;
+
+class CalloutRuleFactory implements CalloutRuleFactoryInterface
+{
+    public function createHasTaxon(array $taxons): CalloutRuleInterface
```

Comment:
What is the purpose of this method? It seems rather specific

Reply:
I needed this method for tests, the similar method has a slise for the promotion rule